OpenChange/MAPIStoreMailVolatileMessage.m

```diff
@@ -533,6 +491,48 @@ - (void) getMessageData: (struct mapistore_message **) dataPtr
   return result;
 }
 
+static inline NSString *
```

Comment:
`MakeRecipientString` has been moved below `QuoteSpecials` so it can return `QuoteSpecials(fullEmail)`. This makes OL display recipients correctly without any other change (SOGo webmail, however, keeps displaying the encoding).